css-cascade/source-order.css

```diff
@@ -0,0 +1,7 @@
+.first-class {
```

Comment:
you are missing the `other-class` here

Reply:
![Screen Shot 2021-10-05 at 9 48 16 PM (2)](https://user-images.githubusercontent.com/67080126/136142725-21028e44-087c-42e2-8622-7a11a2f06dcc.png)
![Screen Shot 2021-10-05 at 9 48 10 PM (2)](https://user-images.githubusercontent.com/67080126/136142731-dc3aefb9-004a-43b9-938a-5a88ec18ce1a.png)

https://user-images.githubusercontent.com/67080126/136142744-fd29f021-672d-4b0c-924d-e3212917fcc0.mov